### ISSUE TITLE
`extra-container destroy` has no effect?

### ISSUE THREAD

**Issue by bjornfor** [erikarvstedt/extra-container#29]
```
$ extra-container list
demo
$ extra-container destroy demo
$ extra-container list
demo                                   <---- still here!?
$ extra-container destroy --all
$ extra-container list
demo                                   <---- still here!?
```

Either I'm misunderstanding the purpose of `destroy` or it's a bug :-) (I thought it'd stop the containers and wipe `/var/lib/nixos-containers/NAME`.)

Using latest extra-container version (0.10).

**Comment by erikarvstedt** [comment]
I can't reproduce this, neither locally, nor by running your example in the reference VM (via `nix run github:erikarvstedt/extra-container#vm`).
What's your system environment? Do you use NixOS? If yes, what's your NixOS version and `system.stateVersion`?

**Comment by bjornfor** [comment]
This is on Ubuntu 18.04. Thanks for checking. I'll try to debug this from my side then.

**Comment by bjornfor** [comment]
With `set -x` debugging I found that it was PEBKAC:

* A long time ago I configured extra-container (at commit 66c8c3d4480e9cb076a966cd7b71fc672f0dfbb6) in my env, which I deployed to a few machines. I ended up not really using it back then and forgot that I had it (inside some home-manager modules).
* Fast forward 1+ years and I set up a new repo with a shell.nix using modern `extra-container`.
* Things work fine and I start deploying to remote machines, not realizing that on the remote machines I'm actually using the *old extra-container* and not the one in my shell.nix file. (Technically the deploy script was correct, but I started running `extra-container` outside of the shell.nix file on the remote machine -- big mistake -- inadvertantly using an old version!)
* The old extra-container version hardcodes /etc/container path, whereas the modern one detects /etc/nixos-containers (when applicable). Thus the old version doesn't see the containers and silently aborts `extra-container destroy`.

So yeah... completely PEBKAC!

Sorry for the noise and thank you for creating extra-container! I'm really exited to have access to declarative NixOS on impure Ubuntu!
